### PR TITLE
feat(HVNC): add presence

### PR DESCRIPTION
**Pull request by nayumiDEV**
**Describe the changes in this pull request:**
add presence support for 
hentai.fakku.cc
moe.hensoe.top
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

image

image

image

image

</details>

### CODE REVIEW

File: websites/H/HVNC/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	},
+	"service": "HVNC",
+	"description": {
+		"en": "HVNC is a free, high-quality, ad-free, uncensored English R18 Manga reading site.",
+		"vi_VN": "HVNC là trang đọc truyện Manga R18 tiếng Anh miễn phí, chất lượng cao, không có quảng cáo, không bị kiểm duyệt."
+	},
+	"url": ["hentai.fakku.cc", "moe.hensoe.top"],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/VlB7zdS.png",
+	"thumbnail": "https://i.imgur.com/FU4jAl1.png",
```
Comment (EncryptedDev) [inline websites/H/HVNC/dist/metadata.json:15]: Censor pornographic content

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/H/HVNC/dist/metadata.json (hunk 1 above)
Comment (ndbiaw) [reply websites/H/HVNC/dist/metadata.json]: done, thannks

File: websites/H/HVNC/presence.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
+const presence = new Presence({
+		clientId: "941317056589086730"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const inPath = document.location.pathname,
+		select = document.querySelector,
```
Comment (EncryptedDev) [inline websites/H/HVNC/presence.ts:8]: Lets not use this _function_ and call document each time

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/H/HVNC/presence.ts (hunk 2 above)
Comment (ndbiaw) [reply websites/H/HVNC/presence.ts:8]: done, thanks

**Review by EncryptedDev — APPROVED** [review]
Looks fine

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**